eng/pipelines/coreclr/crossgen2.yml

```diff
@@ -38,6 +38,7 @@ extends:
           - CoreClrTestBuildHost # Either osx_x64 or linux_x64
           jobParameters:
             testGroup: innerloop
+            hotColdSplitting: true
```

Comment:
We will want later on (in this PR) to add another job. We don't want to get rid of our default Crossgen2 pipelines.

Reply:
Gotcha, while this PR is a draft, I'll start with enabling hot/cold splitting for the default Crossgen2 pipeline. Once that's working, I'll move this logic to a separate job.